Userland/Libraries/LibWeb/SVG/SVGDecodedImageData.cpp

```diff
@@ -181,7 +181,7 @@ Optional<CSSPixelFraction> SVGDecodedImageData::intrinsic_aspect_ratio() const
     // https://www.w3.org/TR/SVG2/coords.html#SizingSVGInCSS
     auto width = intrinsic_width();
     auto height = intrinsic_height();
-    if (width.has_value() && height.has_value())
+    if (width.has_value() && height.has_value() && *height != 0)
```

Comment:
spec mentioned above says that if both width and height has values we should use them to calculate aspect ration. so it seems that we should rather return `{}` in case height is zero instead of falling back to subsequent steps of the algorithm.